packages/app/src/app/components/CreateSandbox/Import/AccountSelect.tsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import styled from 'styled-components';
+import VisuallyHidden from '@reach/visually-hidden';
+import {
+  useMenuState,
+  Menu,
+  MenuItem,
+  MenuDisclosure,
+  MenuSeparator,
+} from 'reakit/Menu';
+
+import { Stack, Text, Icon, Element } from '@codesandbox/components';
+import { useActions } from 'app/overmind';
+
+// In the new codebase this should be a Radix dropdown menu
+// https://www.radix-ui.com/docs/primitives/components/dropdown-menu
+// also, more composable if it were a design system component
+export const AccountSelect = ({ options, value, onChange }) => {
+  const { signInGithubClicked } = useActions();
+  const menu = useMenuState();
+
+  return (
+    <>
+      <StyledMenuButton {...menu}>
+        <Stack gap={2} align="center">
+          <Icon name="github" />
+          <Stack gap={2} align="center">
+            <VisuallyHidden>Select GitHub account.</VisuallyHidden>
+            <Text size={13}>
+              {value} <VisuallyHidden>is currently selected</VisuallyHidden>
+            </Text>
+            <Icon name="chevronDown" size={8} />
+          </Stack>
+        </Stack>
+      </StyledMenuButton>
+      <StyledMenu {...menu} aria-label="GitHub account selection">
+        <Stack direction="vertical">
+          <Element paddingY={2} paddingX={4}>
+            <Text variant="muted">GitHub accounts</Text>
+          </Element>
+          {options.map(option => (
+            <StyledMenuItem
+              {...menu}
+              onClick={() => {
+                onChange(option.login);
+                menu.hide();
+              }}
+              isSelected={option.login === value}
+            >
+              <Stack justify="space-between">
+                <Stack gap={2} align="center" css={{ overflow: 'hidden' }}>
+                  <Icon name="github" />
+                  <Text truncate>{option.login}</Text>
+                </Stack>
+                {option.login === value ? <Icon name="simpleCheck" /> : null}
+              </Stack>
+            </StyledMenuItem>
+          ))}
+          <Element paddingY={1}>
+            <StyledMenuSeparator {...menu} />
+          </Element>
+          <StyledMenuItem
+            {...menu}
+            onClick={() => {
+              signInGithubClicked('private_repos');
+              menu.hide();
+            }}
+          >
+            <Stack direction="vertical" gap={1}>
+              <Text>Review GitHub permissions</Text>
+              <Text variant="muted">
+                Only permitted organizations are visible. Some permission
```

Comment:
Maybe "Only **authorized** organizations..." is more accurate?

Reply:
Good one, changed in [c74210f](https://github.com/codesandbox/codesandbox-client/pull/7436/commits/c74210f29b8dbd3bbf78d31e2b389a70b33b90cc).